### PR TITLE
Add edraak_social and edraak_misc

### PR DESCRIPTION
**Pull request by shadinaif**
This is to add `edraak_social` and `edraak_misc` apps. This will also include changes made by:

https://github.com/Edraak/edx-platform/pull/303 #OER-229 Get course completion status from edx lms
https://github.com/Edraak/edx-platform/pull/331 facebook authentication updated
https://github.com/Edraak/edx-platform/pull/332 set facebook auth_type to the needed value to refresh expired tokens

### CODE REVIEW

**Comment by shadinaif** [comment]
@devalih @OmarEdraak I couldn't find `all_courses.html` file in the theme. Is this thing deprecated?

See the code in the file here https://github.com/Edraak/edraak-platform/blob/35bcb3de4e49177c3164a6b88fa280184f376d1c/lms/templates/all_courses.html

File: lms/djangoapps/edraak_misc/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        view_full_name = 'edraak_misc:{view}'.format(view=view)
+        url = reverse(view_full_name, args=['edx/Demo/2019'])
+
+        res = self.client.get(url)
+        self.assertRedirects(res, '/login?next={}'.format(url))
+
+    @ddt.data(
+        ('check_student_grades', True, '"success": '),
+        ('course_complete_status', True, '"complete": '),
+        ('all_courses', False, ''),
```
Comment (ghost) [inline lms/djangoapps/edraak_misc/tests/test_views.py]: Please remove the `all_courses` view, it's deprecated.

```suggestion
```

File: lms/djangoapps/edraak_misc/views.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    """
+
+    return JsonResponse({
+        'complete': is_student_pass(request.user, course_id)
+    })
+
+
+@ensure_csrf_cookie
+@cache_if_anonymous()
+def all_courses(request, extra_context={}, user=AnonymousUser()):
```
Comment (ghost) [inline lms/djangoapps/edraak_misc/views.py]: Please remove this.

**Review by ghost — APPROVED** [review]

File: lms/djangoapps/edraak_misc/tests/test_views.py (hunk 1 above)
Comment (shadinaif) [reply lms/djangoapps/edraak_misc/tests/test_views.py]: Done

File: lms/djangoapps/edraak_misc/views.py (hunk 2 above)
Comment (shadinaif) [reply lms/djangoapps/edraak_misc/views.py]: Done

**Pull request merged**